---------------------
PatchSet 10314 
Date: 2006/04/28 10:09:58
Author: hno
Branch: SQUID_2_5
Tag: (none) 
Log:
Current release is STABLE13, not 12..

Members: 
	ChangeLog:1.520.2.124->1.520.2.125 

Index: squid/ChangeLog
===================================================================
RCS file: /cvsroot/squid/squid/ChangeLog,v
retrieving revision 1.520.2.124
retrieving revision 1.520.2.125
diff -u -r1.520.2.124 -r1.520.2.125
--- squid/ChangeLog	15 Mar 2006 11:27:50 -0000	1.520.2.124
+++ squid/ChangeLog	28 Apr 2006 10:09:58 -0000	1.520.2.125
@@ -1,4 +1,4 @@
-Changes to squid-2.5.STABLE12 (12 Mar 2006)
+Changes to squid-2.5.STABLE13 (12 Mar 2006)
 	- [Minor] Fails to compile on Solaris and some other platforms
 	  with undefined reference to setenv (Bug #1435)
 	- [Cosmetic] Added WebDAV REPORT method to know HTTP methods list
